tests/foreman/cli/test_report.py

```diff
@@ -84,7 +85,9 @@ def test_positive_install_configure_host(
     """
     puppet_infra_host = [session_puppet_enabled_sat, session_puppet_enabled_capsule]
     for client, puppet_proxy in zip(content_hosts, puppet_infra_host, strict=True):
-        client.configure_puppet(proxy_hostname=puppet_proxy.hostname)
+        client.configure_puppet(
```

Comment:
Is (client_repo_ver == '7') correct here and below?

Reply:
updated, good catch